Packs/AzureActiveDirectory/Integrations/AzureADIdentityProtection/AzureADIdentityProtection.py

```diff
@@ -311,6 +311,60 @@ def build_filter(last_fetch, params):
     return query_filter
 
 
+# TODO: move to CommonServerPython?
+from datetime import datetime, timedelta
+from typing import Callable, Dict
+def fetch_last_run(fetch_func: Callable,
+                   params,
+                   time_format_func: Callable = None,
+                   integration_name=''):
+    '''
+      An abstraction for the lastRun fetch flow.
+
+      Args:
+        fetch_func (Callable): User supplied function to fetch incidents,
+          uses last time as a parameter and returns the new time.
+        params (Dict[str, str]): Demisto configuration object
+        time_format_func (Callable): A user defined function to convert date
+                            time into the intergration specified date format.
+        integration_name (str): Integration name as prefix for debug logs.
+      Returns:
+        None
+    '''
+
+    log_prefix = f'[{integration_name}]' if integration_name else '[Debug]'
```

Comment:
I'd keep the [Debug] log level, and add the integration_name, for standardization.